TypeCobol.Analysis.Test/BasicCfgInstrs/PerformThru2.dot

```diff
@@ -0,0 +1,97 @@
+﻿digraph Cfg {
+compound=true;
+node [
+shape = "record"
+]
+
+edge [
+arrowtail = "empty"
+]
+Block0 [
+label = "{START|}"
+]
+Block1 [
+label = "{Block1|    PERFORM A THRU B\l}"
+]
+subgraph cluster_1{
+color = blue;
+label = "A";
+Block12 [
+label = "{A. Block12|    display \"A\"\l}"
+]
+Block13 [
+label = "{B. Block13|    display \"B\"\l}"
+]
+Block12 -> Block13
+
+}
+Block1 -> Block12 [style=dashed]
+
+Block2 [
+label = "{Block2|}"
+]
+Block3 [
+label = "{Block3|    PERFORM B THRU A\l}"
+]
+subgraph cluster_2{
+color = blue;
+}
+Block3 -> "" [style=dashed]
+
+Block4 [
+label = "{Block4|}"
+]
+Block5 [
+label = "{Block5|    PERFORM S1 THRU S2\l}"
+]
+subgraph cluster_3{
+color = blue;
+label = "S1";
+Block14 [
+label = "{S1. Block14|    display \"S1\"\l}"
+]
+Block15 [
+label = "{S2. Block15|    display \"S2\"\l}"
+]
+Block14 -> Block15
+
+}
+Block5 -> Block14 [style=dashed]
+
+Block6 [
+label = "{Block6|}"
+]
+Block7 [
+label = "{Block7|    PERFORM S2 THRU S1\l}"
+]
+subgraph cluster_4{
+color = blue;
+}
+Block7 -> "" [style=dashed]
+
+Block8 [
+label = "{Block8|}"
+]
+Block9 [
+label = "{Block9|    PERFORM S1 THRU A\l}"
+]
+subgraph cluster_5{
+color = blue;
+}
+Block9 -> "" [style=dashed]
+
+Block10 [
+label = "{Block10|    GOBACK\l}"
+]
+Block0 -> Block1
+Block1 -> Block2
+Block2 -> Block3
+Block3 -> Block4
+Block4 -> Block5
+Block5 -> Block6
+Block6 -> Block7
+Block7 -> Block8
+Block8 -> Block9
+Block9 -> Block10
+
+}
```

Comment:
Is it normal that there's no Block11 declared?

Reply:
I don't get the meaning of your question 😕 
During parsing, the CFG builder will create new block each time a new sentence is encountered and also when branching is required. After building, blocks may be cloned if they participate in a PERFORM target group. When generating the dot file however, only blocks connected to the root are considered. Some blocks created during parsing may finally not appear in the graph.
Do you think the resulting graph is wrong/missing some elements ?

Reply:
I was checking the tests from the .dot files directly, I was wondering about the numbering as I couldn't see any block11 when block10 and block12 were declared.